raquet/geotiff2raquet.py

```diff
@@ -35,6 +35,14 @@
 import pyarrow.parquet
 import quadbin
 
+try:
+    import numpy
+    import numpy.ma
+except ImportError:
+    HAS_NUMPY = False
```

Comment:
Suggestion: Could we include a log message here warning the user about performance issues when using the library without Numpy? If the user is working with medium or large rasters, it might be helpful to understand why Numpy is necessary